cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -237,6 +263,8 @@ async def run_sweep_async(
         program_labels: Optional[Dict[str, str]] = None,
         job_description: Optional[str] = None,
         job_labels: Optional[Dict[str, str]] = None,
+        run_name: str = "",
```

Comment:
Do we want to update calibration methods?

Reply:
We don't. Note that these arguments are for run_sweep, not run_calibration (github view is a bit confusing).